src/components/cc-toggle/cc-toggle.js

```diff
@@ -6,6 +6,7 @@ import { dispatchCustomEvent } from '../../lib/events.js';
 
 /**
  * @typedef {import('./cc-toggle.types.js').Choice} Choice
+ * @typedef {import('../../lib/events.types.js').EventWithTarget<HTMLInputElement>} EventWithTargetHTMLInputElement
```

Comment:
nitpick: I use to name this kind of type like `HTMLInputElementEvent`, but do whatever you prefer

Reply:
let's do that :+1: